snapshot-labs/snapshot
# fix: add missing core tx service to osnap plugin

### Summary

This is follow-up PR from #4575 to add missing Core safe transaction service endpoint to support balance fetching in oSnap transaction builder.

### How to test

1. Add core treasury to safe snap enabled space
2. Start creating proposal with osnap selecting core chain safe
3. Add transactions in transaction builder using `Send tokens` option and check that safe balances are shown there.
